R/Lab 1.Rmd

```diff
@@ -0,0 +1,197 @@
+---
```

Comment:
Is this a duplicate of the one in the root folder of the package? See previous comment for how we should organize this stuff.